pydoctor/__init__.py

```diff
@@ -1,5 +1,9 @@
 """PyDoctor, an API documentation generator for Python libraries."""
 
+__all__ = ["__version__"]
+
 from ._version import __version__
 
-__all__ = ["__version__"]
+
+def _setuptools_version():
+    return __version__.public()
```

Comment:
It would be good to have type annotations on all new code: (even if it's as isolated as this fragment)
```diff
-def _setuptools_version():
-    return __version__.public()
+def _setuptools_version() -> str:
+    return __version__.public() # type: ignore[no-any-return]
```